README.md

```diff
@@ -11,6 +11,7 @@ building and bundling.
 | --- | --- | --- |
 | [ffi](pkgs/ffi/) | Utilities for working with Foreign Function Interface (FFI) code. | [![pub package](https://img.shields.io/pub/v/ffi.svg)](https://pub.dev/packages/ffi) |
 | [ffigen](pkgs/ffigen/) | Generator for FFI bindings, using LibClang to parse C, Objective-C, and Swift files. | [![pub package](https://img.shields.io/pub/v/ffigen.svg)](https://pub.dev/packages/ffigen) |
+| [objective_c](pkgs/objective_c/) | A library to access Objective C from Dart and Flutter that acts as a support library for package:ffigen. | [![pub package](https://img.shields.io/pub/v/objective_c.svg)](https://pub.dev/packages/objective_c) |
```

Comment:
Maybe for now "from Flutter" as this doesn't work from Dart standalone as long as it's a plugin.

Also please file an issue to move over to native-assets once the experimental flag for that is removed.

Reply:
Done